client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ContactsTabsStyles.ts

```diff
@@ -0,0 +1,32 @@
+import { makeStyles } from '@material-ui/styles';
+import {Theme} from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
```

Comment:
I know it is the same but why not importing the theme like in any other useStyle?

Reply:
what do you mean?

Reply:
as in the glboal theme?

Reply:
![image](https://user-images.githubusercontent.com/52863381/100547324-db2ec980-326e-11eb-9a8d-0cc503c3c2ac.png)
like in here

Reply:
that would make it less generic since it uses the global theme instead of the theme wrapping the component that uses the useStyles